lib/actions.js

```diff
@@ -37,5 +37,23 @@ module.exports = class Actions {
         return err ? reply(`An error has occured, \n ${err}`) : reply(data)
       })
     })
+
+    io.action('get unassigned shards', (reply) => {
+      this.es.cat.shards({
+        h: [ 'index', 'state', 'unassigned.reason' ]
+      }, (err, data) => {
+        return err
+          ? reply(`An error has occured, \n ${err}`)
+          : reply(data.split('\n').filter(line => line.indexOf('UNASSIGNED') !== -1).join('\n'))
+      })
+    })
+
+    io.action('get biggers indices', (reply) => {
+      this.es.cat.indices({
+        s: [ 'docs.count:desc' ]
+      }, (err, data) => {
+        return err ? reply(`An error has occured, \n ${err}`) : reply(data)
```

Comment:
Maybe we can get an ouput more like that:

```json
{
	"<index>": {
       "docs": 0,
       "size": "0gb",
       "<type>": {
          "docs": 0,
          "size": "0gb"
       }
    }
}
```

You can get types with `/<index>/_mappings` and stats by type via `/<index>/_stats?types=<type>`